docs/network-drivers.md

```diff
@@ -1,258 +1,319 @@
-# Network Drivers
+	# Network Drivers
```

Comment:
Looks like you have put everything in a code block. which is why everything is indented and the whole file is shown in diff.